sdk/eventhubs/azure_messaging_eventhubs/tests/round_trip.rs

```diff
@@ -0,0 +1,169 @@
+// Copyright (c) Microsoft Corp. All Rights Reserved.
+//cspell: words eventhubs eventhub eventdata amqp
+
+#![cfg(all(test, feature = "test_e2e"))] // to run this, do: `cargo test --features test_e2e`
+
+use async_std::stream::StreamExt;
+use azure_core_amqp::{
+    messaging::{AmqpMessage, AmqpMessageProperties},
+    value::{AmqpList, AmqpValue},
+};
+use azure_identity::{DefaultAzureCredential, TokenCredentialOptions};
+use azure_messaging_eventhubs::{
+    consumer::{ConsumerClient, ConsumerClientOptions, ReceiveOptions},
+    models::{EventData, MessageId, StartPosition},
+    producer::{batch::EventDataBatchOptions, ProducerClient, ProducerClientOptions},
+};
+use futures::pin_mut;
+use std::env;
+use tracing::info;
+
+mod common;
+
+#[tokio::test]
+async fn test_round_trip_batch() {
+    const EVENTHUB_PARTITION: &str = "1";
+    common::setup();
+    let host = env::var("EVENTHUBS_HOST").unwrap();
+    let eventhub = env::var("EVENTHUB_NAME").unwrap();
+    let producer = ProducerClient::new(
+        host.clone(),
+        eventhub.clone(),
+        DefaultAzureCredential::create(TokenCredentialOptions::default()).unwrap(),
+        ProducerClientOptions::builder()
+            .with_application_id("test_new")
```

Comment:
Does this value need to be unique for each test to avoid resource collisions?

Reply:
It's used to identify the connection to the service, which shows up in diagnostics, it does not need to be unique.